src/styles/partials/components/_search.scss

```diff
@@ -0,0 +1,23 @@
+.dg_search-container {
+  position: relative;
+  width: 100%;
+
+  .dg_search-btn {
```

Comment:
You can remove the following unchecked properties.

![image](https://user-images.githubusercontent.com/1933400/64615637-f5019980-d3a8-11e9-8311-6b262938d442.png)

Reply:
deleted them